packages/devtools_app/lib/src/screens/debugger/debugger_controller.dart

```diff
@@ -256,7 +256,8 @@ class DebuggerController extends DisposableController
 
     if (script != null) {
       try {
-        final positions = await getBreakablePositions(script);
+        final positions =
+            await getBreakablePositions(script) as List<SourcePosition>;
```

Comment:
have we verified that this code does not throw? @polina-c ran into some issues where using `as NewType` instead of `.cast<NewType>()` was causing issues. Not sure if this is the same situation or not.

Reply:
Removed the cast!